assets/server/realmadmin/_form_sms.html

```diff
@@ -503,7 +544,11 @@
 
       let $textArea = $newNode.find('textarea');
       $textArea.attr('name', `sms_text_template_${next}`);
-      $textArea.val('Your Exposure Notifications verification link: [enslink] Expires in [longexpires] hours (click for mobile device only)');
+      if (enxEnabled) {
```

Comment:
If we push this into the realm, this gets easier I think

Reply:
done.